_layouts/base.html

```diff
@@ -62,6 +62,7 @@
           </li>
           <li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
           <li><a href="{{ '/policies/privacy-policy.html' | relative_url }}">Privacy Policy</a></li>
+          <li><a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">CC Attrib-SA 4.0 License</a>.</li>
```

Comment:
Looks like a copy/paste issue, but `<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">` is duplicated here. For future reference, would you prefer if I added a quick change via the GitHub web UI here as another commit or would you prefer to eg fix and squash yourself?

Reply:
Fixed!